browser/components/MarkdownEditor.js

```diff
@@ -13,12 +14,18 @@ class MarkdownEditor extends React.Component {
     this.state = {
       status: 'PREVIEW',
       renderValue: props.value,
-      keyPressed: {}
+      keyPressed: {},
+      locked: false
```

Comment:
This is duplicated and not connected state with https://github.com/asmsuechan/Boostnote/blob/74ee6ae6ce047ae7c349b35ae048ae14930c547e/browser/main/Detail/MarkdownNoteDetail.js#L29.